custom_components/meross_cloud/cover.py

```diff
@@ -130,6 +130,10 @@ def supported_features(self):
         # So we don't support that on HA neither.
         return CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE | CoverEntityFeature.STOP
 
+    @property
+    def current_cover_position(self):
+        return self._device.get_position(channel=self._channel_id) or 50
```

Comment:
Why are we returning 50% in case get_position returns None?
[According to the parent Cover class implementation](https://github.com/home-assistant/core/blob/6eaf3402c6534d6db5d9d7157c79bf6246f00fe9/homeassistant/components/cover/__init__.py#L281-L287), we should instead return None, which means "unknown".

Reply:
`None` is treated as closed and `up` button is disabled, at least with `50` both buttons are enabled